lng/german.lng.php

```diff
@@ -684,12 +684,12 @@
 // ADDED IN 1.2.19-svn8
 
 $lng['admin']['dkimsettings'] = 'DomainKey-Einstellungen';
-$lng['dkim']['dkim_prefix']['title'] = 'Prefix';
-$lng['dkim']['dkim_prefix']['description'] = 'Wie lautet der Pfad zu den DKIM-RSA-Dateien sowie den Einstellungsdateien des Milter-Plugins?';
+$lng['dkim']['dkim_prefix']['title'] = 'DKIM-Konfig';
```

Comment:
See english language, also please do not use abbreviations like "Konfig"

Reply:
I will change